src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Handlers/RetryHandler.cs

```diff
@@ -56,15 +56,15 @@ public class RetryHandler : DelegatingHandler
         protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
         {
             HttpResponseMessage response = null;
+
             for (int attempt = 0; attempt < RetryHandler.MaxAttempts; ++attempt)
```

Comment:
This is not very readable, IMHO. We should just do this:
for(int attempt = 1; attempt <= RetryHandler.MaxAttempts; ++attempt)
{
..
}